tests/operators/test_python.py

```diff
@@ -59,6 +59,10 @@
 from tests.test_utils import AIRFLOW_MAIN_FOLDER
 from tests.test_utils.db import clear_db_runs
 
+if TYPE_CHECKING:
+    from airflow.models.dagrun import DagRun
+
+TI = TaskInstance
```

Comment:
Should we unify how `TaskInstance` is referenced in this test?

Reply:
I think we could do as follow up. In some tests `TI` uses as shorthand for `TaskInstance` and I'm not sure should we do that, because it might lead to situation as we have in this test where `TaskInstance` uses for type annotation inside of the tests and `TI` uses for create object 🙄